Fix rendering of Tiled object layers on isometric maps

lfarroco opened the pull request:
This PR:

* Fixes a bug

Describe the changes below:
Objects contained in object layers generated by Tiled use orthogonal positioning even when the map is isometric. In the pictures below, note that `A` has `{x:128, y:128}` and `B` has `{x:256, y:128}` - the `y` is the same, as one would expect in an orthogonal grid (but not in an isometric one).

![image](https://user-images.githubusercontent.com/4733061/135769107-703a3b5a-816f-4ce1-9436-95d0e3233a01.png)
![image](https://user-images.githubusercontent.com/4733061/135769120-5637be46-6efe-495a-a9b2-33155978f944.png)

The current way to position sprites on `createFromObjects` uses that position provided by Tiled and applies an origin offset over it - leading to issues [as described here](https://phaser.discourse.group/t/objects-position-are-wrong-in-isometric-map-created-with-tiled/8771).

This PR converts the sprites' position from cartesian to isometric before applying the offset, positioning elements over the grid as expected.

lfarroco committed:
Change positioning for isometric object layer from Tiled

lfarroco committed:
Revert "Update copy-to-examples.js"

This reverts commit d844f48e870e9b9c95104aa164fa4c2826252442.

lfarroco committed:
Revert "Revert "Update copy-to-examples.js""

This reverts commit 0bec01304791dbf6a0ae4a32391b5317a4fff138.

lfarroco committed:
Revert "Change positioning for isometric object layer from Tiled"

This reverts commit 33f242a2e08de52eb462268e84a3f8afa2d3406a.

lfarroco committed:
Change positioning for isometric object layer from Tiled

lfarroco committed:
linting

photonstorm commented:
Thanks for the PR - can you please confirm this PR will work correctly with the current changes already made to `createFromObjects` as part of 3.60?

lfarroco committed:
adds isometric ratio

lfarroco commented:
Hey! I installed locally from `master` and added the changes, it works as expected.
I also added a ratio calculation, because the original PR assumed a ratio of tile width/height of 2, but that can vary.

photonstorm commented:
Thanks 👍

lfarroco commented:
Thanks for running such an amazing project, @photonstorm !
